provisioner/terraform/provision_test.go

```diff
@@ -22,24 +22,7 @@ import (
 	"github.com/coder/coder/provisionersdk/proto"
 )
 
-func TestProvision(t *testing.T) {
-	t.Parallel()
-
-	provider := `
-terraform {
-	required_providers {
-		coder = {
-			source = "coder/coder"
-			version = "0.4.2"
-		}
-	}
-}
-
-provider "coder" {
-}
-	`
-	t.Log(provider)
-
+func testProvisioner(t *testing.T) (context.Context, proto.DRPCProvisionerClient) {
```

Comment:
I might call this `setupProvisioner` instead. This could give a false impression that this is a test.